Description: use insert instead of add header
Author: A. Schulze
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
Index: smf-spf-2.0.2/smf-spf.c
===================================================================
--- smf-spf-2.0.2.orig/smf-spf.c	2012-09-07 10:19:44.000000000 +0200
+++ smf-spf-2.0.2/smf-spf.c	2012-09-07 10:19:46.000000000 +0200
@@ -843,7 +843,7 @@
 			authserv_id, "none", context->from, context->helo);
 		    break;
 	    }
-	    smfi_addheader(ctx, "Authentication-Results", spf_hdr);
+	    smfi_insheader(ctx, 1, "Authentication-Results", spf_hdr);
 	    free(spf_hdr);
 	}
     }
